addons/website_sale_stock/i18n/pt_BR.po

```diff
@@ -181,24 +187,37 @@ msgid "Show availability Qty"
 msgstr "Mostrar Quant disponível"
 
 #. module: website_sale_stock
+#. odoo-python
 #: code:addons/website_sale_stock/models/sale_order.py:0
 #, python-format
```

Comment:
You can also remove this one
https://github.com/odoo/odoo/blob/eab9453a0dff21fb3672a05eed346e073563a5a6/odoo/tools/translate.py#L811-L813